editor/src/components/canvas/controls/comment-indicator.tsx

```diff
@@ -101,7 +103,19 @@ function CommentIndicatorInner() {
                   boxShadow: UtopiaTheme.panelStyles.shadows.medium,
                 }}
               >
-                {initials}
+                {avatar == null ? (
+                  <span>{initials}</span>
+                ) : (
+                  <img
```

Comment:
can you add an error handler like the one in `user-bar.tsx`? e.g. google login avatars tend to produce a lot of 403s for some reason, falling back to initials makes more sense than showing the broken image placeholder

Reply:
we could create an ImgWithFallback component that wraps around Federico's error handler

Reply:
I made something similar, but just Avatar specific, called AvatarPicture